### DISCUSSION TITLE
Live Camera Stream to ChromeCast

### DISCUSSION THREAD

**Discussion by Asfand-Hafeez** [HaishinKit/HaishinKit.swift#1506] [Q&A]
I am working on screen mirroring app where i need to chromecast live camera on smart with flip front and back camera 
if anyone worked on it please guide me

**Comment by Asfand-Hafeez** [comment]
@shogo4405  kindly respond to this question

**Reply by shogo4405** [reply]
I'm not sure. I don't have a Chromecast.
